projects/aws/lib/aws-stack.ts

```diff
@@ -224,8 +224,15 @@ export class EditionsStack extends cdk.Stack {
                             resources: [
                                 `arn:aws:s3:::editions-store-${lowerCaseStageParameter.valueAsString}/*`,
                                 `arn:aws:s3:::editions-proof-${lowerCaseStageParameter.valueAsString}/*`,
+                                `arn:aws:s3:::editions-proof-${lowerCaseStageParameter.valueAsString}`,
+                                `arn:aws:s3:::editions-store-${lowerCaseStageParameter.valueAsString}`,
+                            ],
+                            actions: [
+                                's3:PutObject',
+                                's3:PutObjectAcl',
+                                's3:ListBucket',
+                                's3:GetObject',
```

Comment:
these new permissions are needed so that we can 'list' the contents of the proof/publish buckets to see how many issues are in there

Reply:
Instead of accessing the bucket directly, would we get the same result by accessing the issues endpoint? I guess its down to what resembles the source of truth.

Reply:
Good point, treating 'issues' as the source of truth might make more sense. Doing it this way is a bit easier though, as it saves having to parse the issues json file, and means we can use the same 'list directories' function in both places where we clean the list.